AWS/aws/s3/trigger_s3_records.py

```diff
@@ -12,6 +12,9 @@ def is_valid_payload(payload: dict) -> bool:
 
         # If eventSource is "s3" then we should collect logs only if the event name starts with one of the following
         if event_source == "s3.amazonaws.com":
+            if payload.get("eventName", "").startswith("GetObjectTagging"):
```

Comment:
Why did you use the startswith and not equal ?

Reply:
Make sense! Thank you. Fixed